features/built_in_matchers/equality.feature

```diff
@@ -92,6 +92,8 @@ Feature: Equality matchers
   Scenario: compare using equal (equal?)
     Given a file named "compare_using_equal.rb" with:
       """ruby
+      # frozen_string_literal: false
```

Comment:
Why?

Reply:
As far as I know, `# frozen_string_literal: false` makes the old ruby behaviors for string literals.
Recent projects basically prefers `# frozen_string_literal: true`.  Then ruby handles same string literal as a same object.

So clarifying to set `false` does not make confusion when reading docs as a tutorial. I think.

Reply:
We use `.feature` files for two purposes:

1. Documentation, that is published to https://relishapp.com/rspec/
I'd prefer not to show an example that the usage of `frozen_string_literal` makes sense for spec files, as I haven't seen any evidence of that.

2. End-to-end tests of our code
Is there a good reason to explicitly specify the already default non-frozen string literal in our own tests?

Reply:
Thanks for letting me know the `cucumber` role in this project.

Personally I don’t have any problem. So I will left from this issue!